.github/workflows/build-packages.yml

```diff
@@ -137,6 +137,19 @@ jobs:
           submodules: 'true'
       - uses: hdl/conda-ci@master
 
+  #9
+  openfpgaloader-linux:
+    runs-on: "ubuntu-16.04"
+    needs: ["libusb-linux", "libftdi-linux"]
```

Comment:
maybe `hidapi` to have access to CMSIS-DAP and orbtrace ?

Reply:
This required adding `libhidapi` build in separate PR (https://github.com/hdl/conda-prog/pull/24). It should be merged before `OpenFPGALoader`. After that I will rebase this branch onto newest master and it should be ok.